src/terra-dev-site/doc/application/reference.1/components/ApplicationNavigation.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import PropsTable from 'terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
+import ApplicationNavigationExample from 'terra-application/lib/terra-dev-site/doc/application/reference.1/components/example/ApplicationNavigationExample?dev-site-example';
+
+# ApplicationNavigation
+
+The ApplicationNavigation component provides a responsive layout and a variety of controls for application navigation.
+
+## Usage
+
+```jsx
+import ApplicationNavigation from 'terra-application/lib/application-navigation';
+```
+
+## Props
+
+<PropsTable />
+
+## Example
+
+<ApplicationNavigationExample />
+
+## Features
+
+### Exception Handling
+
+ApplicationNavigation renders an [ApplicationErrorBoundary](/components/terra-application/application/reference/components/application-error-boundary) around its children. If the children throw exceptions during React lifecycle functions, ApplicationBase will catch the exceptions and render a styled error component in place of the children. 
+
+This error component will render below ApplicationNavigation's header, leaving the user with full access to all navigation controls after an error occurs.
```

Comment:
This makes me think, can the settings controls be used to update the ApplicationNavigation's children content? If so, are these disabled when an error is thrown or how should these be managed by the implementer?

Reply:
I touched on this a bit above: https://github.com/cerner/terra-application/pull/5/files#r355158502

Settings from ApplicationNavigation should application-level, not content-level. Changing settings shouldn't be blocked by a component runtime error caught by the boundary.